packages/backend/src/open_payments/auth/middleware.test.ts

```diff
@@ -32,11 +36,86 @@ describe('Auth Middleware', (): void => {
   let appContainer: TestContainer
   let authServerIntrospectionUrl: URL
   let middleware: AppMiddleware
-  let ctx: PaymentPointerContext
+  let ctx: HttpSigContext
   let next: jest.MockedFunction<() => Promise<void>>
   let validateRequest: ValidateFunction<IntrospectionBody>
   let grantReferenceService: GrantReferenceService
+  let mockKeyInfo: KeyInfo
   const token = 'OS9M2PMHKUR64TB8N6BW7OZB8CDFONP219RP1LT0'
+  let generatedKeyPair: {
+    keyId: string
+    publicKey: JWKWithRequired
+    privateKey: JWKWithRequired
+  }
+  let requestPath: string
+  let requestAuthorization: string
+  let requestBody: Body
+  let requestUrl: string
+  let requestMethod: RequestMethod
```

Comment:
It seems like these could all be defined here as `const` (instead of below in `beforeAll`